src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -48,15 +52,13 @@ public int size() {
         return lottoNumbers;
     }
```

Comment:
해당 코멘트와 같이 고민해보시면 좋을 것 같아요 :)
특히 LottoTicket 같은 경우가 해당 코멘트에서 설명드린 깊은 복사가 필요해 보이는 상황입니다.
그렇기에 해당 코멘트와 같이 고민해보면 좋을 것 같아요!
```suggestion
        return Collections.unmodifiableList(lottoNumbers);
    }
```